examples/multi-agent-systems/room_mas/configs/Room1.json

```diff
@@ -11,6 +11,7 @@
       "measurement_uncertainty": 0.0001,
       "t_sample": 50,
       "save_results": true,
+      "result_filename": "res.csv",
```

Comment:
I left this on purpose, so we have both cases (result with and without filename) in the examples